XamlControlsGallery/ControlPages/CalendarViewPage.xaml

```diff
@@ -43,17 +43,22 @@
             </local:ControlExample.Options>
```

Comment:
This page still has the old language input (TextBox instead of ComboBox). I assume that change will be easy to merge from master when you check in this PR.

Reply:
I was looking at the wrong branch. ComboBox is working beautifully and language is getting updated in the Xaml sample.